leptos_hotkeys/src/use_hotkeys.rs

```diff
@@ -41,47 +42,58 @@ fn is_hotkey_match(hotkey: &Hotkey, pressed_keyset: &mut HashMap<String, web_sys
 }
 
 pub fn use_hotkeys_scoped(
-    key_combination: String,
-    on_triggered: Callback<()>,
-    scopes: Vec<String>,
+    #[cfg_attr(feature = "ssr", allow(unused_variables))] key_combination: String,
```

Comment:
Change this:
```suggestion
    key_combination: String,
```

Execute `cargo make lint`:

```
error: unused variable: `key_combination`
  --> leptos_hotkeys/src/use_hotkeys.rs:45:5
   |
45 |     key_combination: String,
   |     ^^^^^^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_key_combination`
   |
   = note: `-D unused-variables` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(unused_variables)]`

error: could not compile `leptos_hotkeys` (lib) due to 1 previous error
```

Clippy complains about not used when ssr feature active.

Reply:
Note that is not something that affetcs the code, really is a `clippy()` comment.

Reply:
Sounds good! Everything looks great! Really clever with the std::collections::Hashmap :)